ql/termstructures/volatility/optionlet/constantoptionletvol.hpp

```diff
@@ -35,29 +36,33 @@ namespace QuantLib {
     class ConstantOptionletVolatility : public OptionletVolatilityStructure {
       public:
         //! floating reference date, floating market data
-        ConstantOptionletVolatility(Natural settlementDays,
-                                    const Calendar& cal,
+        ConstantOptionletVolatility(Natural settlementDays, const Calendar &cal,
                                     BusinessDayConvention bdc,
-                                    const Handle<Quote>& volatility,
-                                    const DayCounter& dc);
+                                    const Handle< Quote > &volatility,
```

Comment:
no, that formatting is not coming from my side (I double-checked looking in my old PR), looks strange this `Handle< Quote >` - @mdriesch did you apply some reformatting ?